### PR TITLE
[new release] paf, paf-cohttp and paf-le (0.0.8)

### PR DESCRIPTION
**Pull request by dinosaure**
HTTP/AF and MirageOS

- Project page: <a href="https://github.com/dinosaure/paf-le-chien">https://github.com/dinosaure/paf-le-chien</a>
- Documentation: <a href="https://dinosaure.github.io/paf-le-chien/">https://dinosaure.github.io/paf-le-chien/</a>

##### CHANGES:

* Upgrade to `tcpip.7.0.0` (@dinosaure, dinosaure/paf-le-chien#54)
* Let the user to manipulate the incoming flow (@dinosaure, dinosaure/paf-le-chien#55)

### CODE REVIEW

**Comment by mseri** [comment]
```
# Error: Conflict between the following libraries:
# - "piaf.httpaf" in _build/default/vendor/httpaf/lib
# - "httpaf" in /home/opam/.opam/4.08/lib/httpaf
#   -> required by library "gluten" in /home/opam/.opam/4.08/lib/gluten
#   -> required by library "piaf.h2-lwt" in _build/default/vendor/ocaml-h2/lwt
# Error: Conflict between the following libraries:
# - "piaf.httpaf" in _build/default/vendor/httpaf/lib
# - "httpaf" in /home/opam/.opam/4.08/lib/httpaf
#   -> required by library "gluten" in /home/opam/.opam/4.08/lib/gluten
# Error: Conflict between the following libraries:
# - "piaf.httpaf" in _build/default/vendor/httpaf/lib
# - "httpaf" in /home/opam/.opam/4.08/lib/httpaf
#   -> required by library "gluten" in /home/opam/.opam/4.08/lib/gluten
#   -> required by library "piaf.httpaf-lwt" in
#      _build/default/vendor/httpaf/lwt
# Error: Conflict between the following libraries:
# - "piaf.httpaf" in _build/default/vendor/httpaf/lib
# - "httpaf" in /home/opam/.opam/4.08/lib/httpaf
#   -> required by library "gluten" in /home/opam/.opam/4.08/lib/gluten
#   -> required by library "piaf.h2-lwt" in _build/default/vendor/ocaml-h2/lwt
#   -> required by library "piaf.h2-lwt-unix" in
#      _build/default/vendor/ocaml-h2/lwt-unix
```

File: packages/paf-cohttp/paf-cohttp.0.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "logs"              {with-test}
+  "mirage-crypto-rng" {with-test}
+  "mirage-time-unix"  {with-test}
+  "tcpip"             {with-test & >= "6.0.0"}
+  "uri"               {with-test}
+  "lwt"               {with-test}
+  "astring"           {with-test}
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
```
Comment (mseri) [inline packages/paf-cohttp/paf-cohttp.0.0.8/opam:29]:
```suggestion
run-test: ["dune" "runtest" "-p" name "-j" jobs] {os != "macos"}
```

File: packages/paf/paf.0.0.8/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "alcotest-lwt" {with-test}
+  "bigstringaf" {>= "0.7.0"}
+  "httpaf" {>= "0.7.1"}
+  "h2" {>= "0.7.0"}
+  "faraday" {>= "0.7.2"}
+  "tls" {>= "0.15.0"}
+  "cstruct" {>= "6.0.0"}
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
```
Comment (mseri) [inline packages/paf/paf.0.0.8/opam:35]:
```suggestion
run-test: ["dune" "runtest" "-p" name "-j" jobs] { os = "macos" }
```

**Comment by mseri** [comment]
macos sandbox is stricter when trying to use the network

**Comment by dinosaure** [comment]
> macos sandbox is stricter when trying to use the network

This is what I discovered 😕 .

**Pull request merged**